tests/testthat/_snaps/dm.md

```diff
@@ -271,4 +271,102 @@
       $ Species      <fct> setosa, setosa, setosa, setosa, setosa, setosa, setosa, s~
       $ other_col    <int> 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, ~
       $ one_more_col <dbl> 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, ~
+    Code
+      dm_nycflights13() %>% dm_select_tbl(weather) %>% dm_select(weather, -origin) %>%
+        glimpse()
+    Output
+      dm of 1 tables: `weather`
+      
+      Table: `weather`
+      Rows: 144
+      Columns: 14
+      $ year       <int> 2013, 2013, 2013, 2013, 2013, 2013, 2013, 2013, 2013, 2013,~
+      $ month      <int> 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,~
+      $ day        <int> 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10,~
+      $ hour       <int> 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 1~
+      $ temp       <dbl> 41.00, 39.02, 39.02, 39.92, 41.00, 41.00, 39.92, 41.00, 42.~
+      $ dewp       <dbl> 32.00, 30.02, 28.94, 26.96, 26.06, 26.06, 24.98, 24.98, 24.~
+      $ humid      <dbl> 70.08, 69.86, 66.85, 59.50, 54.97, 54.97, 54.81, 52.56, 48.~
+      $ wind_dir   <dbl> 230, 210, 230, 270, 320, 300, 280, 330, 330, 320, 320, 330,~
+      $ wind_speed <dbl> 8.05546, 9.20624, 6.90468, 5.75390, 6.90468, 12.65858, 6.90~
+      $ wind_gust  <dbl> NA, NA, NA, NA, NA, 20.71404, 17.26170, NA, NA, 26.46794, N~
+      $ precip     <dbl> 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,~
+      $ pressure   <dbl> 1024.6, 1025.9, 1026.9, 1027.5, 1028.2, 1029.0, 1030.0, 103~
+      $ visib      <dbl> 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10,~
+      $ time_hour  <dttm> 2013-01-10 00:00:00, 2013-01-10 01:00:00, 2013-01-10 02:00~
+
+# glimpse.zoomed_dm() works
+
+    Code
+      dm_nycflights13() %>% dm_zoom_to(airports) %>% glimpse()
+    Output
+      dm of 5 tables: `airlines`, `airports`, `flights`, `planes`, `weather`
+      
+      Table: `airports`
+      Primary key: (`faa`)
```

Comment:
Should we use parens only for compound keys (which are pretty rare)?

```suggestion
      Primary key: `faa`
```

Reply:
Good idea!

I have implemented this, both for primary and foreign keys (LHS and RHS entries).